plugins/RocketsPlugin/RocketsPlugin.cpp

```diff
@@ -47,7 +47,6 @@
 
 namespace
 {
-const std::string ENDPOINT_API_VERSION = "v1/";
 const std::string ENDPOINT_ANIMATION_PARAMS = "animation-parameters";
```

Comment:
Let's group these methods/endpoints according to which can be a request or a notification. Maybe also according to which have `set-*`/`get-*`.

Reply:
Done